requirements.txt

```diff
@@ -31,6 +31,7 @@ pypet==0.6.1
 pyquaternion==0.9.9
 python-dateutil==2.8.2
 pytz==2022.7.1
+rich==12.6.0
```

Comment:
any reason to not use the latest version?
```suggestion
rich==13.3.2
```

Reply:
![image](https://user-images.githubusercontent.com/3520445/226964209-d9f27d51-20a3-4d26-b417-02c820327e38.png)

Propblem with pygout

Reply:
thats not our dependency though thats just something you use :)

Reply:
Okay, tested and working on 13.2.2